projects/valtimo/exact-plugin/src/lib/exact-refresh-token.ts

```diff
@@ -0,0 +1,6 @@
+export default interface ExactTokens {
+  accessToken: String;
```

Comment:
just to make sure you're using capitalized 'String' intentionally because you want the type to be an object